server/terminal/screen.go

```diff
@@ -62,6 +62,7 @@ func (s *screen) up(i string) {
 // Move the cursor down
 func (s *screen) down(i string) {
 	s.y += pi(i)
+	s.growScreenHeight()
```

Comment:
did you mean to put these `screen.go` modifications in the follow-up PR?

Reply:
removed these, they were intended to fix the new line problem and did not succeed.